frontend/src/pages/projects/notebook/useRouteForNotebook.ts

```diff
@@ -13,15 +13,22 @@ const useRouteForNotebook = (
   const routeNamespace = notebook.metadata.namespace;
 
   React.useEffect(() => {
-    getRoute(routeName, routeNamespace)
-      .then((route) => {
-        setRoute(`https://${route.spec.host}/notebook/${routeNamespace}/${routeName}`);
-        setLoaded(true);
-      })
-      .catch((e) => {
-        setLoadError(e);
-        setLoaded(true);
-      });
+    let watchHandle;
+    const watchRoute = () => {
+      setLoaded(false);
+      getRoute(routeName, routeNamespace)
+        .then((route) => {
+          setRoute(`https://${route.spec.host}/notebook/${routeNamespace}/${routeName}`);
+          setLoaded(true);
+        })
+        .catch((e) => {
+          setLoadError(e);
+          setLoaded(true);
+          watchHandle = setTimeout(watchRoute, 1000);
+        });
```

Comment:
```suggestion
      getRoute(routeName, routeNamespace)
        .then((route) => {
          setRoute(`https://${route.spec.host}/notebook/${routeNamespace}/${routeName}`);
          setLoaded(true);
        })
        .catch((e) => {
          setLoadError(e);
          watchHandle = setTimeout(watchRoute, 1000);
        });
```
Let's just do away with the loaded state flipping -- it's goofy to go back and forth like this.